samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,307 @@
+    [-1735776000000, -0.11],
+    [-1767312000000, -0.1395],
+    [-1767312000000, -0.16],
+    [-1798848000000, -0.3162],
+    [-1798848000000, -0.34],
+    [-1830470400000, -0.3288],
+    [-1830470400000, -0.35],
+    [-1862006400000, -0.4332],
+    [-1862006400000, -0.44],
+    [-1893542400000, -0.3789],
+    [-1893542400000, -0.42],
+    [-1925078400000, -0.4261],
+    [-1925078400000, -0.47],
+    [-1956700800000, -0.4396],
+    [-1956700800000, -0.43],
+    [-1988236800000, -0.3706],
+    [-1988236800000, -0.4],
+    [-2019772800000, -0.2174],
+    [-2019772800000, -0.23],
+    [-2051308800000, -0.2931],
+    [-2051308800000, -0.28],
+    [-2082931200000, -0.4194],
+    [-2082931200000, -0.44],
+    [-2114467200000, -0.3369],
+    [-2114467200000, -0.35],
+    [-2146003200000, -0.2463],
+    [-2146003200000, -0.27],
+    [-2177539200000, -0.1417],
+    [-2177539200000, -0.15],
+    [-2209075200000, -0.0679],
+    [-2209075200000, -0.09],
+    [-2240611200000, -0.1173],
+    [-2240611200000, -0.16],
+    [-2272147200000, -0.2546],
+    [-2272147200000, -0.28],
+    [-2303683200000, -0.1224],
+    [-2303683200000, -0.11],
+    [-2335305600000, -0.0974],
+    [-2335305600000, -0.15],
+    [-2366841600000, -0.229],
+    [-2366841600000, -0.21],
+    [-2398377600000, -0.2808],
+    [-2398377600000, -0.31],
+    [-2429913600000, -0.3212],
+    [-2429913600000, -0.3],
+    [-2461536000000, -0.3062],
+    [-2461536000000, -0.27],
+    [-2493072000000, -0.2532],
+    [-2493072000000, -0.24],
+    [-2524608000000, -0.322],
+    [-2524608000000, -0.37],
+    [-2556144000000, -0.0982],
+    [-2556144000000, -0.12],
+    [-2587766400000, -0.1471],
+    [-2587766400000, -0.2],
+    [-2619302400000, -0.2489],
+    [-2619302400000, -0.33],
+    [-2650838400000, -0.2003],
+    [-2650838400000, -0.31],
+    [-2682374400000, -0.2125],
+    [-2682374400000, -0.32],
+    [-2713996800000, -0.2009],
+    [-2713996800000, -0.28],
+    [-2745532800000, -0.1424],
+    [-2745532800000, -0.21],
+    [-2777068800000, -0.0648],
+    [-2777068800000, -0.1],
+    [-2808604800000, -0.0628],
+    [-2808604800000, -0.12],
+    [-2840227200000, -0.1148],
+    [-2840227200000, -0.2]
+];
+
+Highcharts.chart('container', {
+    title: {
+        text: 'Global temperature mean anomalies from 1880 to 2016'
+    },
+    accessibility: {
+        description: `
+            A chart displaying monthly mean temperature anomalies from 1880 to 2016.
+            It employs two series, one column and one line,
+            to convey the same data at different resolutions.
+            There is only one column per thirty years,
+            but the line series have a datapoint per year.
+        `
+    },
+    series: [{
+        type: 'column',
+        data: data.map((d, i) => (i % 30 === 0) && d),
+        pointWidth: 30
+    }, {
+        data: data,
+        pointInterval: 36e5
+    }],
+    yAxis: {
+        tickInterval: 0.5,
+        endOnTick: true,
+        max: 1
+    },
```

Comment:
What do you think about removing the title of the y-axis? If you think it's important, the y-axis title should be specified imo.